src/pages/textmemo.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState} from 'react';
+import "codemirror/lib/codemirror.css";
+import "@toast-ui/editor/dist/toastui-editor.css";
+
+import { Editor } from "@toast-ui/react-editor";
+
+function WritePage() {
+
+  const editorRef:any = React.createRef();
```

Comment:
```suggestion
  const editorRef: any = React.createRef();
```

띄어쓰기 수정 부탁드립니다~ 
그리고 가능하시다면 any 수정해보는것도 좋을거같아요